### ISSUE TITLE
Direct access to https://ngx-rocket.com/home give a 404

### ISSUE THREAD

**Issue by ctranxuan** [ngx-rocket/ngx-rocket.github.io#1]
If you copy-paste the link in a browser: 

https://ngx-rocket.com/home

this gives a 404 github page. While going to https://ngx-rocket.com is fine (it strangely redirects to https://ngx-rocket.com/home but that seems ok).

Tested on Linux with Chrome 67 and Firefox 60.0.2

**Comment by sinedied** [comment]
Yes, it's a bug we need to address as github doesn't support PWA routing.
The simplest fix is to enable legacy hashbang routing, @gaetanmaisse could you take a look?
